imageregistry/v1/00-crd.yaml

```diff
@@ -154,6 +154,11 @@ spec:
                   type: object
                   additionalProperties:
                     type: string
+            rolloutStrategy:
+              description: rolloutStrategy defines rollout strategy for the image
+                registry deployment. Valid values are Rolling (default) and Recreate.
+              type: string
+              pattern: ^(Rolling|Recreate)$
```

Comment:
This should align with the types used in k8s deployments: `pattern: ^(RollingUpdate|Recreate)$`

https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L335-L343

Reply:
done!